src/payment/strategies/braintree/braintree-script-loader.ts

```diff
@@ -13,7 +13,7 @@ export default class BraintreeScriptLoader {
 
     loadClient(): Promise<BraintreeClientCreator> {
         return this._scriptLoader
-            .loadScript('//js.braintreegateway.com/web/3.70.0/js/client.min.js')
+            .loadScript('//js.braintreegateway.com/web/3.81.0/js/client.min.js')
```

Comment:
Just one think out of curiosity, does it make sense to make the verison  a constant? So we don't need to change in every place?

`const version = '3.81.0';`
and the can use it like
```
`//js.braintreegateway.com/web/${version}/js/client.min.js`
```

Reply:
fixed